mattermost/mattermost
# [MM-56660] When deleting a draft , the modal delete button does not have focus

#### Ticket Link
Fixes: https://github.com/mattermost/mattermost/issues/26115
Jira: [MM-56660](https://mattermost.atlassian.net/browse/MM-56660)


#### Screenshots
<img width="952" alt="Current behavior" src="https://github.com/mattermost/mattermost/assets/22616979/6a6c386f-6bfd-40c2-a7d3-cea89aa03463">
<img width="947" alt="Expected behavior" src="https://github.com/mattermost/mattermost/assets/22616979/41ed62c2-7506-4a5e-8d6f-0ea5c0a5350b">


#### QA-Notes
Please refer screenshots. The draft should be deleted when pressed **Enter**.


```
NONE
```




[MM-56660]: https://mattermost.atlassian.net/browse/MM-56660?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
